debs/SPECS/wazuh-agent/debian/postinst

```diff
@@ -95,8 +95,9 @@ case "$1" in
         fi
 
         # Set correct permissions, owner and group
-        chmod 640 ${DIR}/ruleset/sca/*
-        chown root:${GROUP} ${DIR}/ruleset/sca/*
+        # There may not be any sca rules, so avoid wildcards
+        chmod -R u=rwX,g=rX,o= ${DIR}/ruleset/sca/
+        chown -R root:${GROUP} ${DIR}/ruleset/sca/
```

Comment:
If we accept this change, the script will not run `chmod` to the files inside the directory, please modify it so that the `chmod` command is applied only if the directory has content, also these changes should be applied in Wazuh manager as well

https://github.com/wazuh/wazuh-packages/blob/6085928b0ec99e38a39b8244f55c945b1d3fbbda/debs/SPECS/wazuh-manager/debian/postinst#L226

Reply:
The -R flag is --recursive, it applies the chmod rule to the directory and all files inside the directory.

The X flag means that it applies execute to the directory, but not the files.

Functionally this change shouldn't have any impact on any existing working installation.

Using recursive, rather than the wildcard, does the content check for us and avoids any other potential nasty effects of wildcard expansion. (Wildcard expansion can lead to other issues like when there are stupid numbers of files and you have more command arguments than are supported. I don't think that is an issue here, but a general rule of thumb is that using recursive flags is safer than wildcards.)

Reply:
Thanks, I added a change for the wazuh-manager